account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you also add a testcase for SNS Bank MT940 files, just to be sure besides Rabobank & ING. See attachment for an example file. 

[tr-info_15785279_20180104123832.zip](https://github.com/OCA/bank-statement-import/files/1603485/tr-info_15785279_20180104123832.zip)

And even documentation (Dutch too bad) if needed:
[6_0016_35_1511_Informatiewijzer_MT940_rapportage.pdf](https://github.com/OCA/bank-statement-import/files/1603487/6_0016_35_1511_Informatiewijzer_MT940_rapportage.pdf)

If the test fails on some SNS specific thing, let me know I can help translating to English if needed.

Ps. I made the same remark on v11 PR: https://github.com/OCA/bank-statement-import/pull/137/files/39e7b8871db5f23bc35013318bc60ba3c8f5b5bb#diff-5bea8377b2408bd073f1a0f6508bd10b